saxon.py

```diff
@@ -9,44 +9,71 @@
 class Saxon:
 
     def __init__(self, saxon_path=SAXON_PATH, math2svg_path=MATH2SVG_PATH):
-        math2svg_path = os.path.abspath(math2svg_path)
-        saxon_path = os.path.abspath(saxon_path)
+        math2svg_path = abspath(math2svg_path)
+        saxon_path = abspath(saxon_path)
+        wrapper_file_path = join(
+            dirname(saxon_path), "SaxonTransformWrapper.java")
+
+        if not isfile(wrapper_file_path):
+            raise IOError("File: {} not found".format(wrapper_file_path))
+        if not isfile(saxon_path):
+            raise IOError("File: {} not found".format(saxon_path))
+        if not isfile(math2svg_path):
+            raise IOError("File: {} not found".format(math2svg_path))
 
-        if not os.path.exists(saxon_path):
-             raise IOError("File: {} not found".format(saxon_path))
-        if not os.path.isfile(math2svg_path):
-             raise IOError("File: {} not found".format(math2svg_path))
+        try:
+            subprocess.check_output("which javac".split())
+        except subprocess.CalledProcessError as e:
+            raise RuntimeError("'javac' command not found.  "
+                               "Try running 'apt-get install openjdk-7-jdk'.")
```

Comment:
a bit system specific on this error message. How about "Please install an appropriate JDK (>= 7)"